### PR TITLE
Several improvements and code fix

### PR DESCRIPTION
**Pull request by ogarcia**
- Better README.md.
- Added setup.py for an easy installation in a virtualenv.
- Updated for use python-magic pip version.
- Fixed a print typo.
- Fixed a small issue with Python3 (startswith expect binary).
- Now server sends a `y` instead a `w` to start file transfer.

### CODE REVIEW

File: fileteasend/app.py [hunk 1]
```diff
@@ -0,0 +1,169 @@
+#! /usr/bin/env python
```
Comment (brechin) [inline fileteasend/app.py]: It doesn't make sense for this file to be executed directly, so I would remove the shebang line.

File: fileteasend/app.py [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+import magic
+import requests
+
+class FileTeaClient(object):
+    """Base client class for FileTea access"""
+    def __init__(self, url=None):
+        logger = logging.getLogger('filetea.FileTeaClient')
+        # Set URL to default if not URL is passed
+        self.url = 'https://filetea.me/' if url == None else url
```
Comment (brechin) [inline fileteasend/app.py]: Comparisons with None should be done with `is`, i.e. `... if url is None else url`

File: fileteasend/app.py [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+        send_url = urljoin(self.url, 'transport/lp/send')
+        send_data = {
+                "method": "addFileSources",
+                "params": [[
+                    file_name,
+                    mime_type,
+                    file_size
+                          ]],
+                "id": "1"
+                    }
```
Comment (brechin) [inline fileteasend/app.py]: Style nitpick: I would prefer this either at the end of the previous line or aligned at character 9 with the `s` below.

File: fileteasend/app.py [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+    parser.add_argument('file', help='file to send')
+    parser.set_defaults(func=run)
+    args = parser.parse_args()
+
+    # Set loglevel equivalents
+    log_levels = {
+            0: logging.ERROR,
+            1: logging.INFO,
+            2: logging.DEBUG
+                 }
```
Comment (brechin) [inline fileteasend/app.py]: Same style nitpick as above, I'd prefer this closing bracket to align with the main indentation level, not the opening bracket.

**Review by brechin — CHANGES_REQUESTED** [review]
Óscar, thanks for the excellent contribution! This is a great improvement to the docs, structure, and function of the code.

Feel free to ignore the style nitpicks.

Please consider adding a CONTRIBUTORS file to recognize your contributions.

File: fileteasend/app.py (hunk 1 above)
Comment (ogarcia) [reply fileteasend/app.py]: Yez, this is caused by a template :smile:

**Comment by ogarcia** [comment]
Changes done!
If you add a version tag after the merge I can make a "stable" package for Arch Linux :wink:

**Review by brechin — APPROVED** [review]

**Comment by ogarcia** [comment]
Perfect!

The package is in AUR: https://aur.archlinux.org/packages/fileteasend

Thanks!

**Pull request merged**